data/mods/Aftershock/items/books.json

```diff
@@ -17,6 +17,38 @@
     "chapters": 30,
     "fun": 6
   },
+  {
+    "id": "afs_welcome1_book",
+    "type": "BOOK",
+    "name": { "str": "Unofficial History of Salus IV Vol. 1", "str_pl": "copies of Unofficial History of Salus IV Vol. 1" },
+    "description": "Welcome to Frozen Hell, an Unofficial History of Salus IV by Bette Lapin.  Salus IV was founded in the early days of the Hyperspace Era by the Interstellar Soviet Community.  At it's height several hundred years ago Salus IV had just over one billion sentient citizens, but that population had begun to decline even before the Discontinuity as people migrated to planets that while more hostile to life during the earliest colonization phases than Salus IV were much simpler to terraform into a place more enjoyable by human and uplift citizens.",
+    "weight": "700 g",
+    "volume": "750 ml",
+    "price": 5000,
+    "price_postapoc": 1000,
+    "looks_like": "ZSG",
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "light_blue",
+    "intelligence": 4,
+    "time": 20,
+    "chapters": 3,
+    "fun": 2
+  },
+  {
+    "id": "afs_welcome2_book",
+    "type": "BOOK",
+    "name": { "str": "Unofficial History of Salus IV Vol. 2", "str_pl": "copies of Unofficial History of Salus IV Vol. 2" },
+    "description": "Even during the Discontinuity Salus IV's population continued to drop as there were easier places to live even in the Salus system, from honeycombed asteroid habitats, to domed cities with gorgeous views.  As with many other governments during the Discontinuity the Interstellar Soviet collapsed into a vipers pit of microstates, solar corporations and other forms of self governance.   During this time Mercurial Genomics a corporation specializing in Uplifts succesfully laid claim to Salus IV in the court system set up by ESCA.  The Whately family, which runs Mercurial Genomics begins leasing development and exploitation permits and allows the establishment of minor ESCA outposts within Salus IV.  There is an official citizen count of less than fifty thousand sentient beings mostly in orbit of Salus IV.   It is believed however that there are at least that many non-citizen sentients spread amongst squatter, smuggler, pirate, and feral uplift communities.",
```

Comment:
```suggestion
    "description": "Even during the Discontinuity Salus IV's population continued to drop as there were easier places to live even in the Salus system, from honeycombed asteroid habitats, to domed cities with gorgeous views.  As with many other governments during the Discontinuity the Interstellar Soviet collapsed into a vipers pit of microstates, solar corporations and other forms of self governance.   During this time Mercurial Genomics a corporation specializing in Uplifts successfully laid claim to Salus IV in the court system set up by ESCA.  The Whately family, which runs Mercurial Genomics begins leasing development and exploitation permits and allows the establishment of minor UICA outposts within Salus IV.  There is an official citizen count of less than fifty thousand sentient beings mostly in orbit of Salus IV.   It is believed however that there are at least that many non-citizen sentients spread amongst squatter, smuggler, pirate, and feral uplift communities.",
```

Just fixing ESCA here, so far looks good, just calling back to my earlier review, how populated are systems in general? and Salus IV? Is a billion citizens small, or a lot? and going from that much to 50,000 + non-citizens is a big jump, so curious

Reply:
A billion I would have considered a small number for a society that used extensive cloning/vat grown uplifts and sentient robots.   By definition the Discontinuity would have cracked that number instantly by whatever number of sentient robots went mad or simply stopped functioning normally.  Space is huge and humanity tends to explode in population when we encounter new frontiers/ overcome biological impediments such as disease.  https://ourworldindata.org/uploads/2019/06/2019-Revision-%E2%80%93-World-Population-Growth-1700-2100-772x550.png

Look at the growth between 1920 and 2020 and I imagined hundreds of years of similar growth.

Reply:
I think something around the low hundred of millions citizens would be appropriate. It is somewhat low for a whole planet, but it lets you fit some 5 or so Greater NYC sized urban centers into it, and if you assume  Salus was only ever densely urbanized near volcanic hot spots or other places that where  slightly more hospitable, I think this seems relatively believable.

Current population I'd chalk a bit lower too, probably just tens of hundreds.

Reply:
Part of why I went high on the numbers is the idea that if someone plays a reasonably long game of Aftershock they could easily kill several thousand uplift mobs just through murder hoboing through the dungeon areas and I didn't want that to be like "well Jimbob the Space Hunter has eliminated the entire population of Salus IV per the documentations".    So I'm definitely good to adjust numbers down but I'm trying to make it so we don't have that weird situation.